runtime/oti/VMHelpers.hpp

```diff
@@ -2048,13 +2049,68 @@ class VM_VMHelpers
 #endif /* JAVA_SPEC_VERSION > 11 */
 	}
 
+#if JAVA_SPEC_VERSION >= 19
+/*
+ *	If low tagging failed (or not even attempted) due to either
+ *
+ *   a carrier thread winning to mount, we don't need to do anything, since it will be compensated by pre/post mount actions
+ *   another GC thread winning to scan, again don't do anything, and let the winning thread do the work, instead
+ */
+	static VMINLINE bool
+	tryWinningConcurrentGCScan(J9VMContinuation *continuation)
+	{
+		volatile uintptr_t *localAddr = &continuation->carrierThread_concurrentlyScanned;
+		uintptr_t oldValue = *localAddr;
+
+		if (0 == oldValue) {
+			/* atomically low tag ('set', not 'or') continuation->carrierThread_concurrentlyScanned */
+			if (VM_AtomicSupport::lockCompareExchange(localAddr, oldValue, J9_GC_CONTINUATION_CONCURRENTSCANNING) == oldValue) {
+				return true;
+			}
+		}
+		return false;
+	}
+
+	static VMINLINE J9VMThread *
+	getCarrierThreadFromCarrierThreadConcurrentlyScanned(uintptr_t carrierThread_concurrentlyScanned)
```

Comment:
It's good that you introduced this getter, and probably should have something to getConcurrentScannedFlag... (or isConcurrentlyScanned...), too. 

But this is a long, hard to read, name of a method :( Not sure, but perhaps, we could just omit From part ?

Also, still thinking about the name of that variable. It could be even as simple as 'state' (but add a comment at its decalaration line it's a bit-wise struct of CarrierThread ID and ConcurrentlyScanned flag. In that case this method name  could be slightly more sensible like getCarrierThreadFromContinuationState